app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id
+  has_many :likes, foreign_key: :author_id
+  has_many :comments, foreign_key: :author_id
+
+  scope :recent_three, -> { :posts.limit(3).order(created_at: :desc) }
```

Comment:
- [ ] Instead of using `scope`, make sure you just create a new method that returns the 3 most recent posts. Usually, we use `scope` to avoid duplicated code in the project. take a look at how the new method to implement will look here below:
```ruby
def recent_post
  posts.order('created_at Desc').limit(3)
end
```